brightnessctl.c

```diff
@@ -381,12 +386,12 @@ unsigned long percent_to_val(float percent, struct device *d) {
 }
 
 void print_device(struct device *dev) {
-	char *format = p.mach ? "%s,%s,%d,%d%%,%d\n" :
-		"Device '%s' of class '%s':\n\tCurrent brightness: %d (%d%%)\n\tMax brightness: %d\n\n";
+	char *format = p.mach ? "%s,%s,%d,%.4g%%,%d\n" :
+		"Device '%s' of class '%s':\n\tCurrent brightness: %d (%.4g%%)\n\tMax brightness: %d\n\n";
```

Comment:
Looking at https://github.com/Hummer12007/brightnessctl/pull/51#issuecomment-628502839 wouldn't the `--frac` incompatible with the `-m` option ? should this be documented or handled in some way ?

Reply:
Nah, I checked, we don't setlocale, so it always uses the dot